app/routes.js

```diff
@@ -9,14 +9,25 @@ module.exports = function(app) {
         res.sendfile('index.html');
     });
 
-    // get all screen documents
-    app.get('/api/screens', function(req, res) {
+    // get all screen ids
+    app.get('/api/screen_ids', function(req, res) {
         Screen
-          .find({})
-          .exec(function(err, data) {
-            if (err) res.send(err);
-            res.json(data);
-          });
+            .find({})
+            .select('_id')
+            .exec(function(err, data) {
```

Comment:
define a separate function for this since it is repeated at least thrice. (love that word.)